galaxykit/containers.py

```diff
@@ -23,6 +24,19 @@ def delete_container(client, name):
     """
     delete_url = f"_ui/v1/execution-environments/repositories/{name}/"
     return client.delete(delete_url, parse_json=False)
+
+def create_container(client, name, upstream_name, registry):
+    """
+    Create container
+    """
+    create_url = f"_ui/v1/execution-environments/remotes/"
+    registry_id = registries.get_registry_pk(client, registry)
+    data = {
+        'name'              : name,
+        'upstream_name'     : upstream_name,
+        'registry'          : registry_id
```

Comment:
```suggestion
        "name": name,
        "upstream_name": upstream_name,
        "registry": registry_id,
```

to fit the style used in namespaces, users and collections.py